core/src/appleMain/kotlin/CoreBluetoothAdvertisement.kt

```diff
@@ -5,7 +5,7 @@ import platform.Foundation.NSData
 import platform.Foundation.NSUUID
 
 public interface CoreBluetoothAdvertisement : Advertisement {
-    public val identifier: NSUUID
+    public override val identifier: Identifier
```

Comment:
Since `CoreBluetoothAdvertisement`'s `identifier' now has same type as `Advertisement`, we can remove the `override` entirely (as it will be inherited).

```suggestion
```